shared-scripts/angular-linker/esbuild-plugin.mjs

```diff
@@ -48,12 +48,13 @@ async function assertNoPartialDeclaration(filePath, ast, traverseFn) {
  * @param ensureNoPartialDeclaration Whether an additional check ensuring there are
  *   no partial declarations should run.
  */
-export async function createLinkerEsbuildPlugin(filter, ensureNoPartialDeclaration) {
+export async function createLinkerEsbuildPlugin(filter, ensureNoPartialDeclaration, options) {
```

Comment:
Just some nit: Want to name this `linkerOptions` and maybe initialize the default directly here?